pkgs/by-name/ge/gema/package.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchurl
+, installShellFiles
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gema";
+  version = "2.0";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/gema/files/gema/${finalAttrs.pname}-${finalAttrs.version}/${finalAttrs.pname}-${finalAttrs.version}-source.tar.gz";
+    hash = "sha256-c7qxgZYk/QaqocjRXVlgJxUWCgf3T1JBY7v9Fg9YfIU=";
+  };
+
+  makeFlags = [ "--directory=src" ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 src/gema -t $out/bin
+    installManPage doc/gema.1
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "General purpose text processing utility based on the concept of pattern matching. It reads an input file and copies it to an output file transforming the data as specified by the patterns defined by the user.";
```

Maybe shorten the `description` to a single sentence and add more info about the package as needed using `longDescription` with the long description text spanning multiple lines, e.g.:

```suggestion
    description = "General purpose text processing utility based on the concept of pattern matching.";
    longDescription = ''
      Reads an input file and copies it to an output file transforming the
      data as specified by the patterns defined by the user.
    '';
```

---

Done! I also sorted longDescription to between license and mainProgram.